kontrakter/varetekt/helseOgRisiko/1.0/helseOgRisiko.schema.json

```diff
@@ -0,0 +1,392 @@
+          "$ref": "#/definitions/risikovurdering",
+          "description": "Opplysninger av betydning for risikovurdering, skal alltid fylles ut."
+        },
+        "helseInfo": {
+          "$ref": "#/definitions/helseInfo",
+          "description": "helseopplysninger skal alltid fylles ut"
+        },
+        "tilstandArresten": {
+          "$ref": "#/definitions/tilstandArresten",
+          "description": "Tilstand til siktede i arresten, skal alltid fylles ut."
+        }
+      },
+      "required": [
+        "skrevetTid",
+        "risikovurdering",
+        "helseInfo",
+        "tilstandArresten"
+      ],
+      "additionalProperties": false
+    },
+    "risikovurdering": {
+      "type": "object",
+      "description": "Vurdering avkrysning og mulighet for beskrivelse i fritekst",
+      "properties": {
+        "risiko": {
+          "type": "array",
+          "description": "Valg av vilke risikotyper, hvis det er ingen risiko så skal INGEN brukes.",
+          "items": {
+            "$ref": "#/definitions/risikoVurderingTyper"
+          },
+          "minItems": 1
+        },
+        "beskrivelse": {
+          "type": "string"
+        }
+      },
+      "required": ["risiko"],
+      "additionalProperties": false
+    },
+    "risikoVurderingTyper": {
+      "type": "string",
+      "description": "Risikovurderinger",
+      "enum": [
+        "VOLDSHISTORIKK",
+        "PSYKIATRI",
+        "UOENSKET_HENDELSE",
+        "ROEMNINGSFARE",
+        "FARE_ANSLAG",
+        "ANDRE_FORHOLD",
+        "UKJENT",
+        "INGEN"
+      ]
+    },
+    "tilstandArresten": {
+      "type": "object",
+      "description": "Valg av tilstand i arresten med beskrivelse",
+      "properties": {
+        "tilstand": {
+          "type": "array",
+          "description": "Valg av hvilken tilstand",
+          "items": {
+            "$ref": "#/definitions/tilstandArrestenTyper"
+          },
+          "minItems": 1
+        },
+        "beskrivelse": {
+          "type": "string"
+        }
+      },
+      "required": ["tilstand"],
+      "additionalProperties": false
+    },
+    "tilstandArrestenTyper": {
+      "type": "string",
+      "description": "Tilstand i arresten",
+      "enum": [
+        "RUSET",
+        "OPPFOERSEL",
+        "FARE_PSYKISK_FYSISK_HELSE",
+        "SELVMORDSRISIKO",
+        "MEDISINER",
+        "UKJENT",
+        "INGEN"
+      ]
+    },
+    "helseInfo": {
+      "type": "object",
+      "description": "Helsetilstanden til personen i arresten",
+      "properties": {
+        "helse": {
+          "type": "array",
+          "description": "Helseinformasjon, velger ingen hvis det ikke er noe å bemerke.",
+          "items": {
+            "$ref": "#/definitions/helseTyper"
+          },
+          "minItems": 1
+        },
+        "beskrivelse": {
+          "type": "string"
+        }
+      },
+      "required": ["helse"],
+      "additionalProperties": false
+    },
+    "helseTyper": {
```

Comment:
Det er uttrykt ønske fra Kriminalomsorgen om ytterligere kategorier, hvis det er mulig å få med i kontrakten allerede nå - ref. [dokumentasjon her](https://domstol.atlassian.net/wiki/spaces/ESAS/pages/4232019992/Status+-+Bestilling+av+varetektsplass#%C3%98nsker-fra-KDI--09.08.24)

Reply:
Vi kan ikke love ytterligere kategorier til pilot (tid), og vi trenger noen avklaringer på disse:
Kanskje handikap burde erstattes med tilrettelegging eller tilrettelegging_handikap og en god beskrivelse til våre brukere hva som er viktig for Kriminalomsorgen.
Dvs. vi endrer ikke med flere kategorier nå.